driver/driver_default.go

```diff
@@ -13,6 +17,17 @@ type DefaultDriver struct {
 	r Registry
 }
 
+// IsSQLiteMemoryMode returns true if SQLite if configured to use memory mode
+func IsSQLiteMemoryMode(dsn string) bool {
```

Comment:
maybe add an unit test for this function as well?

Reply:
I intended to do exactly that. I rename the file containing the old unit test. Part of the commit is the driver_default_test.go file. Have you seen this?

Reply:
Right, sorry I missed that the fact that the actual bug is already fixed on master